service/service.py

```diff
@@ -100,6 +100,21 @@ def create_product():
 ######################################################################
 # UPDATE AN EXISTING PRODUCT
```

Comment:
@sjy252 I think there is a test case needed for this? The code below I grabbed from Prof. Rofrano's lab-flask-rest repo. I'm still not entirely sure how this is all working...but when I downloaded the branch and ran tests on it they failed. 
```suggestion
# UPDATE AN EXISTING PRODUCT
```
   def test_update_a_pet(self):
        """ Update a Pet """
        pet = PetFactory()
        logging.debug(pet)
        pet.create()
        logging.debug(pet)
        self.assertEqual(pet.id, 1)
        # Change it an save it
        pet.category = "k9"
        original_id = pet.id
        pet.save()
        self.assertEqual(pet.id, original_id)
        self.assertEqual(pet.category, "k9")
        # Fetch it back and make sure the id hasn't changed
        # but the data did change
        pets = Pet.all()
        self.assertEqual(len(pets), 1)
        self.assertEqual(pets[0].id, 1)
        self.assertEqual(pets[0].category, "k9")